test/unit/Fixture/ClassForHinting.php

```diff
@@ -4,4 +4,5 @@
 
 class ClassForHinting
 {
+    public $someProperty;
```

Comment:
This class is likely used somewhere else. Consider using a separate test asset for this particular test scenario